crates/test_utils/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 /// This is intended to be used in tests, so it defaults to DEBUG level.
 pub fn enable_tracing() {
     let _ = tracing_subscriber::fmt()
+        .with_test_writer()
```

Comment:
I missed this on the first pass, but this is super useful. It is documented to understand libtest's `-- --nocapture` behaviour somehow and makes this much nicer to use in tests. You don't get any log output unless you ask for it